testsuite/sclang/TestInteger.sc

```diff
@@ -0,0 +1,29 @@
+TestInteger : UnitTest {
+	test_forby_with_positive_integer_arguments {
```

Comment:
It's not 100% standardized yet, but I've got a WIP document for the project test suite [here](https://gist.github.com/brianlheim/91222d487afa18582c287b0a722ae272). The naming convention I'm trying to adopt is `test_methodName_inputConditions[_expectedResult]`, so following that this test should be named something like `test_forBy_positiveIntegerArguments`. Also, you can omit the messages if you want, it's nice to have but IMO they're not super helpful beyond the names of the test functions.

Reply:
Done, thanks for the pointer to the guide.